### PR TITLE
Purge comments loader completely

### PR DESCRIPTION
**Pull request by pupboss**
Hi, thanks for sharing the theme. Previously I met the same issue, so I am familiar with the codebase :). It seems that you forgot to remove the js script.

Also, you can run `npm audit fix` and `npm update` several times to fix the vulnerable dependency issue.

### CODE REVIEW

**Comment by pupboss** [comment]
And I made some additional changes, such as:

1. open external links in a new tab
2. Add icons ahead of every link in post (see picture)
3. Add more social icons (and renamed the icon name to avoid adblocker. because some icon class attributes are blocked by the blocker plugin)

Let me know if you are interested with any of the feature. I can submit a pull request.

screenshot

File: src/js/post.js [hunk 1]
```diff
@@ -112,7 +112,6 @@ const prepareProgressCircle = () => {
 $(document).ready(() => {
   $aosWrapper = $('.js-aos-wrapper')
   const $scrollButton = $('.js-scrolltop')
-  const $loadComments = $('.js-load-comments')
   const $commentsIframe = $('.js-comments-iframe')
```
Comment (eddiesigner) [inline src/js/post.js:115]: Would you mind to remove this variable as well? :)

Comment (pupboss) [reply src/js/post.js:115]: Haha thanks for the alert

**Comment by eddiesigner** [comment]
Hey, thanks for this PR! I completely forgot about that part 😅, btw, I wrote a comment because there is one more variable which is not used anymore.

About the other points:

1. This is more like a preference I would say, only a couple of people have requested such feature so probably is better to add an entry in the Wiki with the instructions of how to do it if necessary.

2. This is also more like a preference and probably not all the people will benefit from that, I can think in more cases where you don't want to display icons next to social links, also that could be a problem because if you display icons for social links, users might expect to have them for pretty much all the social links you can think of, not only a few and of course we can't provide icons for everything, that's why I think is better if we keep it like it is now.

3. I'm not sure about this either, the idea is to provide just a small set of icons for platforms that are used broadly, I don't want to increase the size of the icon font too much, the goal is not to include a whole set of icons within the theme.

**Comment by pupboss** [comment]
Totally understand. No problem, I have submitted another PR :)

**Pull request closed without merging**